### ISSUE TITLE
Documentation: Instructions overwrite custom_components/life360.py

### ISSUE THREAD

**Issue by bmenchaca** [pnbruckner/homeassistant-config#10]
The instructions state to copy over custom_components/life360.py from life360.py, and then device_tracker/life360.py.  Is this intended?

life360.py at <config>/custom_components/life360.py and
device_tracker/life360.py at <config>/custom_components/life360.py

**Comment by GaryOkie** [comment]
Hey Ben,

You will need to create a /custom_components/device_tracker subdirectory, and then copy the second device_tracker/life360.py to it.   There will then be 2 life360.py sources installed.

I'm not sure why it was done this way (2 different sources with the same name), but Phil must have had a good reason :)

**Comment by pnbruckner** [comment]
D'oh! Copy and paste error. Thanks @bmenchaca for pointing that out!

@GaryOkie, the reason for the two modules is, according to the development guidelines, component and platform code is not supposed to directly interact with an external service. They are supposed to use a "lower level" library/module/package that does that. Normally that lower level package would come from the [Python Package Index](https://pypi.org/). But when I looked, there wasn't one. I did find a basic module I could use elsewhere, which is what I started with (and enhanced somewhat significantly.) Eventually I hope to "officially" add it to the PPI, but that's actually a lot of work. So, for now, I just added it to custom_components.
